packages/matchbox/src/helpers/focus.js

```diff
@@ -0,0 +1,6 @@
+const FOCUSABLE_SELECTOR =
+  'a,frame,iframe,input:not([type=hidden]):not(:disabled),select:not(:disabled),textarea:not(:disabled),button:not(:disabled),*[tabindex]';
+
+export function findFocusableChild(element) {
+  return element.querySelector(FOCUSABLE_SELECTOR);
```

Comment:
Honestly not sure how brittle this is – wondering if we should write tests with different focusable activators.